web/src/features/fireWeather/slices/parseModelValuesSlice.ts

```diff
@@ -0,0 +1,33 @@
+import { ModelRun, ModelValue } from 'api/modelAPI'
+import { isNoonInPST } from 'utils/date'
+
+export const parseModelValuesSlice = (
+  model_runs: ModelRun[],
+  separate_noon_values: boolean
+): Record<string, ModelValue[]> => {
+  const pastModelValues: ModelValue[] = []
+  const modelValues: ModelValue[] = []
+  const noonModelValues: ModelValue[] = []
+  const allModelValues = model_runs.reduce(
+    (values: ModelValue[], modelRun) => values.concat(modelRun.values),
+    []
+  )
+  const currDate = new Date()
+  allModelValues.forEach(v => {
+    const isFutureModel = new Date(v.datetime) >= currDate
+    if (isFutureModel) {
+      modelValues.push(v)
+    } else if (separate_noon_values && isNoonInPST(v.datetime)) {
+      noonModelValues.push(v)
+    } else {
+      pastModelValues.push(v)
+    }
```

Comment:
If you look at the old `modelSlices` the logic is slightly different. We need to always include noon values in the `pastModelValues`